specification/communication/data-plane/JobRouter/preview/2021-10-20-preview2/communicationservicejobrouter.json

```diff
@@ -3405,14 +3211,15 @@
         "kind": {
           "type": "string"
         },
-        "threshold": {
-          "description": "Threshold for wait time for this trigger. Requires input conforming to ISO8601 duration format.",
-          "type": "string"
+        "thresholdSeconds": {
```

Comment:
This would be REST API change. Are you sure you want to change that?

Reply:
@weidongxu-microsoft yes, we would like to go with this. This was a miss from our end during our initial commit. All our other time durations are numeric. The guideline was to move away from ISO 8601 duration format.

Reply:
In this case, you are likely going to need a breaking changes review.

Reply:
I went over the Breaking change checklist, none of them applies to us **yet**. We are trying to go public preview. But we could do a breaking change review if that's required

Reply:
So you've done API review for public review https://github.com/Azure/azure-rest-api-specs/issues/18013, but you never released it for public review?

Reply:
That is correct. The service hasn't been deployed for public preview yet. We are simultaneously doing our SDK review for public preview as well. So this change will only impact the yet-to-be-released SDKs that are currently being developed for public preview.

Reply:
Following the policies for breaking changes, I have created this https://msazure.visualstudio.com/One/_workitems/edit/14958963

Reply:
I see. Then there should be a quick resolution. If you are not urgent, use Monday's office hour, if urgent, you can loop in Jeffrey.

I don't have comments on other changes as they are purely for SDK (if you can ping me the ApiView with SDK arch sign-off).